swagger.yaml

```
@@ -3498,7 +3498,104 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/ignored-cluster-validations:
```

```suggestion
  /v2/clusters/{cluster_id}/ignored-validations:
```

Let's save an API and just specify both cluster and host validations in the body - one JSON for each.
Also, need a GET API to return the current value.